packages/app-plugin-security-cognito/src/identityProvider/cognitoPasswordValidator.ts

```diff
@@ -0,0 +1,49 @@
+import { validation, ValidationError } from "@webiny/validation";
+import { PasswordPolicy } from "../types";
+
+const COGNITO_PASSWORD_POLICY = "cognitoPasswordPolicy";
+
+export const cognitoPasswordValidator = (policy: PasswordPolicy): string => {
+    // Create custom validator
+    const cognitoPasswordValidator = createCognitoPasswordValidator(policy);
+
+    validation.setValidator(COGNITO_PASSWORD_POLICY, cognitoPasswordValidator);
+
+    let validator = COGNITO_PASSWORD_POLICY;
+
+    if (policy.minimumLength !== undefined) {
+        validator += `,minLength:${policy.minimumLength}`;
+    }
+
```

Comment:
The minLength can be included directly in the cognito assword validation function.